src/styles.css

```diff
@@ -158,6 +198,59 @@ nav {
   margin: 15px;
 }
 
+.individual-recipe-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.add-panel {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 20px;
+  height: 4vh;
+  background-color: #43444860;
+  border-radius: 20px;
+  width: 10%;    
+  margin-left: 4px;
+  transition: 0.2s;
+}
+
+.add-panel > h4 {
+  font-size: 19px;
+  border: wheat;  
+  height: 0px;
+  width: 0px;
+  transition: 0.2s;
+  visibility: hidden;
+  opacity: 0;
+  font-weight: 100;
+}
+
+.plus-symbol {
+  display: flexbox;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.add-panel:hover {
+  width: 50%;
+  cursor: pointer;
+  background-color: #66686f;
+}
+
+.add-panel:hover .plus-symbol {
+  display: none;
+}
+
+.add-panel:hover h4 {
+  animation: fade-in 0.2s ease-in forwards;
+  width: auto;
+  height: auto;
+}
```

Comment:
is there a way we can make Add recipes to Cook text stay within it's parent div? It looks to be coming out right now on some recipes. 
![Screen Shot 2023-05-20 at 1 40 09 PM](https://github.com/lauraguerra1/whats-cookin/assets/121131581/bbe76231-1c45-471a-844f-9eeb5eebc133)

Reply:
done